modules/controller/index.js

```diff
@@ -3,30 +3,41 @@ const _ = require('lodash');
 const parseOptions = require('./lib/parse_options');
 const extract = require('./lib/extract');
 
-const createResponse = require('./lib/responders/create');
-const readResponse = require('./lib/responders/read');
-const updateResponse = require('./lib/responders/update');
-const destroyResponse = require('./lib/responders/destroy');
-const lookupFailedResponse = require('./lib/responders/lookup_failed');
-
-const filterInitValidator = require('./lib/validators/init/filters');
-const relationsInitValidator = require('./lib/validators/init/relations');
-const modelMethodInitValidator = require('./lib/validators/init/model_method');
-const relationExistsReqValidator = require('./lib/validators/request/relation_exists');
-const contentTypeReqValidator = require('./lib/validators/request/content_type');
+const payloads = {
+  create: require('./lib/responders/create'),
+  read: require('./lib/responders/read'),
+  readRelation: require('./lib/responders/read'),
```

Comment:
I had something like this at one point. There's some loss here requiring the same module twice, no?